client/app/pages/admin/Jobs.jsx

```diff
@@ -121,14 +120,10 @@ class Jobs extends React.Component {
   }
 }
 
+const JobsPage = withUserSession(Jobs);
```

Comment:
A follow up to previous question is: why wrap it here instead of in the route definition?